### PR TITLE
make supported VCS providers list more prominent and explicit

### PR DESCRIPTION
**Pull request by armchairlinguist**

### CODE REVIEW

File: content/source/docs/enterprise/vcs/index.html.md [hunk 1]
```diff
@@ -14,6 +14,19 @@ Terraform Enterprise (TFE) is more powerful when you integrate it with your vers
 
 We recommend configuring VCS access when first setting up a TFE organization, and you might need to add additional VCS providers later depending on how your organization grows.
 
+## Supported VCS Providers
+
+For details on configuring VCS access, select a supported VCS provider from the list below:
```
Comment (nfagerlund) [inline content/source/docs/enterprise/vcs/index.html.md]:
```suggestion
TFE supports the following VCS providers:
```

Comment (armchairlinguist) [reply content/source/docs/enterprise/vcs/index.html.md]: Hm...I was trying to keep in a note that you can click those links to get detailed instructions. Do you think that's unnecessary?

Comment (armchairlinguist) [reply content/source/docs/enterprise/vcs/index.html.md]: @nfagerlund want to take another look? I changed the intro line to match yours, and added the other info below.

File: content/source/docs/enterprise/vcs/index.html.md [hunk 2]
```diff
@@ -14,6 +14,19 @@ Terraform Enterprise (TFE) is more powerful when you integrate it with your vers
...
+
+- [GitHub](./github.html)
+- [GitHub Enterprise](./github-enterprise.html)
+- [GitLab.com](./gitlab-com.html)
+- [GitLab EE and CE](./gitlab-eece.html)
+- [Bitbucket Cloud](./bitbucket-cloud.html)
+- [Bitbucket Server](./bitbucket-server.html)
+
+Use the links above to see details on configuring VCS access for each supported provider. If you use another VCS that is not supported, you can build an integration via [the API-driven run workflow](../run/api.html).
+
```
Comment (koikonom) [inline content/source/docs/enterprise/vcs/index.html.md]: I'm wondering if we should replace "is not supported" with "is not in the list above" ?

**Pull request merged**